test/functional/cmdLineTests/sigabrtHandlingTest/sigabrtHandlingTest.xml

```diff
@@ -26,73 +26,110 @@
 
 <suite id="SIGABRT Handling Tests" timeout="2400">
  <variable name="CP" value="-cp $Q$$RESJAR$$Q$ VMBench.GPTests.GPTest" />
- 
- <!-- 
+
+ <!--
    The system, JIT, snap and heap dumps are disabled to reduce the memory footprint of
    the tests. Only a Java dump will be generated when the JVM abort handler is invoked.
    This should be sufficient to test the -XX:[+|-]HandleSIGABRT option.
  -->
  <variable name="DUMP" value="-Xdump:system+heap+snap+jit:none" />
+ <variable name="LIMT_HEAPSIZE" value="-Xmx20m" />
+ <variable name="NON_WINDOWS_PLATFORMS" value="aix.*,linux.*,zos.*,osx.*" />
```

Comment:
Can you not say `^win.*` for non-Windows platforms?

Reply:
No other place I could find uses ^. Tried it and it doesn't work.

Reply:
Negation is not supported, unfortunately; see [TestConfigParser](https://github.com/eclipse/openj9/blob/master/test/functional/cmdline_options_tester/src/TestConfigParser.java#L348).

Reply:
Actually, because the value is a comma-separated list of regular expressions, this might work (in a future change):
```
<variable name="NON_WINDOWS_PLATFORMS" value="^(?!(all|win.*))"
```

Reply:
That expression doesn't work, it just skips the tests. `Number of PASSED tests: 5 out of 5`